src/Microsoft.Framework.Configuration/ConfigurationSection.cs

```diff
@@ -65,14 +87,14 @@ public string Value
 
                 foreach (var src in Sources)
                 {
-                    src.Set(Key, value);
+                    src.Set(Path, value);
                 }
             }
         }
 
         protected override string GetPrefix()
         {
-            return _key + Constants.KeyDelimiter;
+            return Path + Constants.KeyDelimiter;
```

Comment:
Name of this method is now a bit confusing. Prehaps `GetChildrenPrefix()` would be better. What do you think?